docs/reference/settings.md

```diff
@@ -197,6 +229,80 @@ WAGTAILADMIN_COMMENTS_ENABLED = False
 
 Sets whether commenting is enabled for pages (`True` by default).
 
+### `WAGTAIL_ALLOW_UNICODE_SLUGS`
+
+```python
+WAGTAIL_ALLOW_UNICODE_SLUGS = True
+```
+
+By default, page slugs can contain any alphanumeric characters, including non-Latin alphabets. Set this to False to limit slugs to ASCII characters.
+
+### `WAGTAIL_AUTO_UPDATE_PREVIEW`
+
+```python
+WAGTAIL_AUTO_UPDATE_PREVIEW = True
+```
+
+When enabled, the preview panel in the page editor is automatically updated on each change. If set to `False`, a refresh button will be shown and the preview is only updated when the button is clicked.
+This behaviour is enabled by default.
+
+### `WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL`
+
+```python
+WAGTAIL_AUTO_UPDATE_PREVIEW_INTERVAL = 500
+```
+
+The interval (in milliseconds) to check for changes made in the page editor before updating the preview. The default value is `500`.
+
+(WAGTAILADMIN_GLOBAL_PAGE_EDIT_LOCK)=
+
+### `WAGTAILADMIN_GLOBAL_PAGE_EDIT_LOCK`
+
+`WAGTAILADMIN_GLOBAL_PAGE_EDIT_LOCK` can be set to `True` to prevent users from editing pages that they have locked.
+
+### `WAGTAILADMIN_UNSAFE_PAGE_DELETION_LIMIT`
+
+```python
+WAGTAILADMIN_UNSAFE_PAGE_DELETION_LIMIT = 20
+```
+
+This setting enables an additional confirmation step when deleting a page with a large number of child pages. If the number of pages is greater than or equal to this limit (10 by default), the user must enter the site name (as defined by `WAGTAIL_SITE_NAME`) to proceed.
+
+(WAGTAILADMIN_RICH_TEXT_EDITORS)=
```

Comment:
I think there should be a new section under page editing - maybe General editing?

It should contain `WAGTAILADMIN_RICH_TEXT_EDITORS`, `WAGTAILADMIN_EXTERNAL_LINK_CONVERSION` and the date.time items. 

`WAGTAIL_DATE_FORMAT`, `WAGTAIL_DATETIME_FORMAT`, `WAGTAIL_TIME_FORMAT`

- these all relate to general form fields and are not specific to page editing.